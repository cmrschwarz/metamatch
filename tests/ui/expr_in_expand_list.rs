use metamatch::metamatch;

fn main() {
    let v = "42";
    metamatch!(match v {
        #[expand(for FOO in [v.as_bytes()])],
        "42" => FOO,
        _ => b""
    });
}
