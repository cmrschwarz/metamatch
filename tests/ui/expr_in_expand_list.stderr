error: syntax error in list: expected comma or `]`
 --> tests/ui/expr_in_expand_list.rs:6:31
  |
6 |         #[expand(for FOO in [v.as_bytes()])],
  |                               ^
